Lusas_Adapter/Lusas_Adapter.csproj

```diff
@@ -73,10 +73,9 @@
     <Reference Include="Geometry_oM">
       <HintPath>..\..\BHoM\Build\Geometry_oM.dll</HintPath>
     </Reference>
-    <Reference Include="Lusas.Interop, Version=15.2.1601.22607, Culture=neutral, PublicKeyToken=ee620220e102840a, processorArchitecture=x86">
-      <SpecificVersion>False</SpecificVersion>
+    <Reference Include="Lusas.Interop, Version=17.0.1601.28524, Culture=neutral, PublicKeyToken=ee620220e102840a, processorArchitecture=x86">
+      <HintPath>..\packages\BHoM.Interop.Lusas170.1.0.0\lib\net45\Lusas.Interop.dll</HintPath>
```

Comment:
It does not appear to be able to push anything and gives the following message:

> 1. This component failed to run properly. Are you sure you have the correct type of inputs?
> Check their description for more details. Here is the error provided by the method:
> A null reference pointer was passed to the stub. (Exception from HRESULT: 0x800706F4)

Reply:
Can you check the version you have for Lusas ?

![image](https://user-images.githubusercontent.com/16853390/61524213-8e1ad080-aa48-11e9-9513-fbd81e43df05.png)

Reply:
Apologies for just getting around to this - version shown below.

![image](https://user-images.githubusercontent.com/30078587/61938286-a3cc5080-af88-11e9-9429-a1a2bafad674.png)